storage/mem.go

```diff
@@ -0,0 +1,63 @@
+package storage
```

Comment:
Implement the storage interfaces backed by a map.

Reply:
Nice.